wordpress-mobile/WordPress-Android
# Media picker tracking

Fixes #12646

This PR contains few fixes:
- it adds tracking to all user interactions with the media picker - all the clicks and the screen view. 
- I've consolidated all the navigation events into one object. I think it makes the code less cluttered. 
- I've added preview to stock library and WP media library options (it was previously missing)

To test:
- Check that the preview works in all 3 data sources
- Go through the flows and check that all user interactions are tracked (using Logcat and the "Tracked" filter)
- As part of ^ check that all the navigation actions work (preview, edit, insert media, click on icons, exit the screen)

PR submission checklist:
